tests/x509/test_x509_ext.py

```diff
@@ -5769,6 +5769,22 @@ def test_public_bytes(self):
 
 
 class TestSignedCertificateTimestamps:
+    def test_strip_scts(self, backend):
+        cert = _load_cert(
+            os.path.join("x509", "cryptography-scts.pem"),
+            x509.load_pem_x509_certificate,
+            backend,
+        )
+
+        expected_tbs_precertificate_bytes = load_vectors_from_file(
+            filename=os.path.join("x509", "cryptography-scts-tbs-precert.der"),
+            loader=lambda data: data.read(),
+            mode="rb",
+        )
+        assert (
+            expected_tbs_precertificate_bytes == cert.tbs_precertificate_bytes
+        )
+
```

Comment:
This should be with the certificate tests -- this test class is for testing the SCTs class.

Reply:
Done, I think -- it's now under the certificate extension tests for the extension that's being stripped out in `tbs_precertificate_bytes`.